src/charts/tooltip.js

```diff
@@ -297,6 +333,11 @@ export default function Tooltip (_container, isLegend = false) {
     return this
   }
 
+  function setData (_data) {
```

Comment:
That would not be needed depending on your answer on my previous comment.

Reply:
right, I was originally going to try to replicate Immerse's decade binning behavior, in which case would need the data object, but then we decided against it. The data object might come in handy in the future for the tooltip which is why I left that method intact. I'm okay with removing it though.